Formula/quex.rb

```diff
@@ -12,7 +12,9 @@ class Quex < Formula
     sha256 "f3d39a7468e8c529ce1c0d6ab5b2d028f50771304993e9f2e996490f846c4b6c" => :high_sierra
   end
 
-  uses_from_macos "python@2"
+  # Migration to Python 3 has started, see for example:
+  # https://sourceforge.net/p/quex/git/ci/e0d9de092751dc0b61e0c7fa2777fdc49ea1d13e/
+  uses_from_macos "python@2" # Does not support Python 3
```

Comment:
Can we upstream the `depends_on :macos` lines from Homebrew/linuxbrew-core for python@2 formulae?

Reply:
I think we can. Please keep the `Due to Python2` comment so it makes clear why it is the case.